ocsp/ocsp.go

```diff
@@ -69,13 +58,21 @@ func (e *Entry) verifyResponse(resp *ocsp.Response) error {
 	if resp.ThisUpdate.After(resp.NextUpdate) {
 		return fmt.Errorf("malformed OCSP response: NextUpdate is before ThisUpate (%s before %s)", resp.NextUpdate, resp.ThisUpdate)
 	}
-	if e.serial.Cmp(resp.SerialNumber) != 0 {
-		return fmt.Errorf("malformed OCSP response: Serial numbers don't match (wanted %s, got %s)", e.serial, resp.SerialNumber)
+	if serial.Cmp(resp.SerialNumber) != 0 {
+		return fmt.Errorf("malformed OCSP response: Serial numbers don't match (wanted %s, got %s)", serial, resp.SerialNumber)
 	}
-	e.info("New response is valid, expires in %s", humanDuration(resp.NextUpdate.Sub(now)))
 	return nil
 }
 
+var statusToString = map[int]string{
```

Comment:
Could this be replaced with the `ResponseStatus` `String()`? https://github.com/golang/crypto/blob/master/ocsp/ocsp.go#L30:L60 ?

I think the only place it was being used is L157 (now commented out)